optimum/intel/openvino/configuration.py

```diff
@@ -39,10 +39,20 @@ class OVQuantizationMethod(str, Enum):
     AWQ = "awq"
 
 
+_DEFAULT_4BIT_CONFIG = {
+    "bits": 4,
+    "ratio": 1.0,
+    "sym": False,
+    "group_size": 128,
+    "all_layers": None,
+}
+
+
 _DEFAULT_4BIT_CONFIGS = {
     "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "ratio": 0.8},
     "EleutherAI/gpt-j-6b": {"bits": 4, "sym": False, "group_size": 64},
     "facebook/opt-6.7b": {"bits": 4, "sym": False, "group_size": 64, "ratio": 0.8},
+    "bigscience/bloomz-7b1": dict(_DEFAULT_4BIT_CONFIG, **{"group_size": 32}),
```

Comment:
Fix after removal of this config in #824. I've updated the default config with `group_size=32`, otherwise compression with `group_size=128` and `ratio=1.0` fails.

cc @KodiaqQ

Reply:
What exactly is it fix? Test or real model failure?

Reply:
@KodiaqQ, both

Reply:
Please provide an example of a failing model with this configuration. My experiments show that **bloomz-7b1** successfully quantized with the `group_size=128`. The test is not the case.

Reply:
@KodiaqQ that's right. I've updated the test case to rely on different model ids. 

As I understand this test is to check that the default config for a specific model is used, and it is not critical that a tiny model is `falcon-40b`, but config for `falcon-7b` is used. I've selected this pair of models because `falcon-7b` has `"group_size"=64` in its default config and tiny `falcon-40b` is one of the few models that actually can be compressed with such group size.

cc @eaidova